src/Microsoft.Kusto.ServiceLayer/LanguageServices/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -0,0 +1,329 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using Kusto.Language;
+using Kusto.Language.Editor;
```

Comment:
Please move this code to the DataSource folder.

Reply:
Moved the folder DataSourceIntellisense to DataSource folder